module/zfs/dbuf.c

```diff
@@ -48,6 +48,96 @@
 #include <sys/callb.h>
 #include <sys/abd.h>
 
+kstat_t *dbuf_ksp;
+
+typedef struct dbuf_stats {
+	/*
+	 * Various statistics about the size of the dbuf cache.
+	 */ 
+	kstat_named_t cache_count;
+	kstat_named_t cache_size;
```

Comment:
Let's rename `cache_size` to `cache_size_bytes` to better fit in with the other `target`, `lowater`, `hiwater`, `max` values.